enterprise_gateway/services/kernels/remotemanager.py

```diff
@@ -412,6 +467,9 @@ def _capture_user_overrides(self, **kwargs):
         of the kernelspec env stanza that would have otherwise overridden the user-provided values.
         """
         env = kwargs.get("env", {})
+        # see if KERNEL_LAUNCH_TIMEOUT was included from user.  If so, override default
+        if env.get("KERNEL_LAUNCH_TIMEOUT", None):
+            self.kernel_launch_timeout = float(env.get("KERNEL_LAUNCH_TIMEOUT"))
```

Comment:
We can scrap the if statement...
```suggestion
        self.kernel_launch_timeout = float(env.get("KERNEL_LAUNCH_TIMEOUT", default_kernel_launch_timeout))
```

Reply:
L398, I have already initialised with the default value. So, I skipped setting this again. 
let me know if we still need to add this here?

Reply:
I understand. It just seems cleaner and more maintainable to not have the extra `if` statement IMO.

Reply:
okay .. ll make the change

Reply:
Gentle ping regarding the suggested change.